app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -613,7 +616,9 @@ class StoryFragmentTest {
   }
 
   @Test
-  @DisableAccessibilityChecks // TODO(#3362): Enable AccessibilityChecks
+  // TODO(#3245): Error -> View falls below the minimum recommended size for touch targets
+  // TODO(#3245): Error -> URLSpan should be used in place of ClickableSpan
```

Comment:
Please mention it as a single TODO with both the errors. 

`// TODO(#3245): Use URLSpan and minimum touch target size should be 48x48 dp`